kernel/kernel-api/src/main/java/io/delta/kernel/internal/data/TransactionStateRow.java

```diff
@@ -133,4 +135,15 @@ public static List<String> getPartitionColumnsList(Row transactionState) {
   public static String getTablePath(Row transactionState) {
     return transactionState.getString(COL_NAME_TO_ORDINAL.get("tablePath"));
   }
+
+  /**
+   * Get the maxRetries from scan state {@link Row} returned by {@link
+   * Transaction#getTransactionState(Engine)}
+   *
+   * @param transactionState Transaction state state {@link Row}
```

Comment:
IMO this is one of those cases where I don't think the `@param` nor `@return` comments offer any value.

The first sentence tells me (a) what this method does, and (b) gives me all the context not in the code that I need to know (that this transaction state came from the getTransactionState call).

(And this is good! This means the code is self describing!)

Reply:
Lol because this is literally what I thought when I was doing this but I just copied the style in the rest of the file